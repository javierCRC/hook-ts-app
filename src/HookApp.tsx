import React from 'react'

export const HookApp = () => {
    return (
        <>
             <h1 className="text-primary"> Hola Mundo TypeScript </h1>
        </>
    )
}

