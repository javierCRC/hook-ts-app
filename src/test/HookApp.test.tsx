import { shallow } from "enzyme"
import { HookApp } from "../HookApp"
import '@testing-library/jest-dom'; 


describe('Pruebas de <HookApp />', () => {
    
    test('Must do to render the component HookApp', () => {
        
        // 1 Arrange *******************************************************************************
      
        //2. Act *******************************************************************************

        const cWrapper = shallow(<HookApp />);
    
        //3. Assert *******************************************************************************

        expect(cWrapper).toMatchSnapshot();

    })
    
})