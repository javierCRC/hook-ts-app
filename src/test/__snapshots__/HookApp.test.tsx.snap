// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas de <HookApp /> Must do to render the component HookApp 1`] = `
<Fragment>
  <h1
    className="text-primary"
  >
     Hola Mundo TypeScript 
  </h1>
</Fragment>
`;
